### PR TITLE
DEVEXP-561 Now supporting REST API transactions

### PR DESCRIPTION
**Pull request by rjrudin**

### CODE REVIEW

File: marklogic/transactions.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        if time_limit:
+            params["timeLimit"] = time_limit
+        if database:
+            params["database"] = database
+
+        response = self._session.post(
+            "/v1/transactions",
+            params=params,
+            headers={"Accept": "application/json"},
+        )
```
Comment (BillFarber) [inline marklogic/transactions.py:97]: Strangely, this doesn't match the documentation at https://docs.marklogic.com/REST/POST/v1/transactions

Comment (rjrudin) [reply marklogic/transactions.py:97]: Do you mean the use of the "Accept" header? I may have eagerly added that, I'll try removing it. 

I'm also going to change the code for when a transaction can't be committed / rolled back, seems like `raise_for_status()` should be used there.

Comment (BillFarber) [reply marklogic/transactions.py:97]: Also that the documentation says to use the "Location" header in the response (which I find strange).

Comment (rjrudin) [reply marklogic/transactions.py:107]: Added comments, the docs are incorrect.

**Review by BillFarber — APPROVED** [review]

**Pull request merged**